### PR TITLE
finmind CREATE

### PR DESCRIPTION
**Pull request by roccqqck**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/finmind/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/FinMind-{{ version }}.tar.gz
+  sha256: f19bc2a7c9698ea80c4a78d2d3eb27daad01c8bec896cc34da9ff8f74a413644
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/finmind/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/finmind/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - python
+
+test:
+  imports:
+    - FinMind
+
+
+about:
+  home: https://github.com/FinMind/FinMind
+  license: MIT
```
Comment (marcelotrevisani) [inline recipes/finmind/meta.yaml]: The license here is ``MIT``, however, you packaged ``Apache-2.0`` (and it is also missing some information in the license file)

File: recipes/finmind/LICENSE [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (marcelotrevisani) [inline recipes/finmind/LICENSE]: It is missing the year and copyright owner here

File: recipes/finmind/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    - python
+
+test:
+  imports:
+    - FinMind
+
+
+about:
+  home: https://github.com/FinMind/FinMind
+  license: Apache 2.0
```
Comment (marcelotrevisani) [inline recipes/finmind/meta.yaml]:
```suggestion
  license: Apache-2.0
```

File: recipes/finmind/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/finmind/meta.yaml]: It seems that this project depends on ``pandas``, ``lxml``, ``requests``, ``numpy``, ``keras``, ``matplotlib``, ``scikit-learn``. Am I right? If so, could you please add them as well?
```suggestion
    - python
    - pandas
    - lxml
    - requests
    - numpy
    - keras
    - matplotlib
    - scikit-learn
```

Comment (roccqqck) [reply recipes/finmind/meta.yaml]: but when i pip install FinMind,
pands and matplotlib are not finmind's dependencies.

i think i should keep it that way.

Comment (roccqqck) [reply recipes/finmind/meta.yaml]: its a package to import some stock datasets as json file.
we dont need to crawl the website.
https://github.com/FinMind/FinMind#what-is-it


https://github.com/FinMind/FinMind#translation-%E4%B8%AD%E8%8B%B1%E5%B0%8D%E7%85%A7
of course we could use pandas to process json dataset from finmind
but we dont really NEED pandas.

**Pull request merged**